common-html/footer.html

```diff
@@ -1,5 +1,6 @@
 <footer>
   <div class="global-footer">
-    <p><img alt="" /></p>
-    <p class=>All content Copyright ニッカオフィス / knicka office</p>
-</footer>
\ No newline at end of file
+    <p class="global-footer-logo"><img src="./img/logo-knicka.png" alt=""></p>
```

Comment:
```suggestion
    <p class="global-footer__logo"><img src="./img/logo-knicka.png" alt=""></p>
```
BEM記法にするため、`logo` は、アンダースコア2個でつなげてください